cli/running/running.go

```diff
@@ -4,29 +4,28 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"os"
 	"os/exec"
 	"path"
 	"path/filepath"
-	"strconv"
 	"strings"
 	"syscall"
 	"time"
 
 	"github.com/tarantool/tt/cli/cmdcontext"
 	"github.com/tarantool/tt/cli/config"
 	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/process_utils"
```

Comment:
Why not just `github.com/tarantool/tt/cli/process`?

Reply:
It is because we have type `Process` in `daemon/process.go`, having `process` package and type `process` in different package causes confusion.